src/Evolution/Ringdown/StrahlkorperCoefsInRingdownDistortedFrame.hpp

```diff
@@ -16,23 +16,33 @@
  */
 namespace evolution::Ringdown {
 /*!
- * \brief Transform `Strahlkorper` coefs to ringdown distorted frame.
+ * \brief Transforms `Strahlkorper` coefs to ringdown distorted frame and tracks
+ * the center of mass.
```

Comment:
Does it track the center of mass, or does it track the geometric center of the AhC?